component/vshn_alerting.jsonnet

```diff
@@ -4,6 +4,69 @@ local inv = kap.inventory();
 local params = inv.parameters.appcat;
 local vars = import 'config/vars.jsonnet';
 
+local getRedisHARules(serviceName) = [
+  {
+    alert: 'VSHNRedisNotMaster',
+    annotations: {
+      description: 'Service {{ $labels.service }} is not connected to a master for 5m.',
+      summary: 'Redis not master: {{ $labels.name }} ({{ $labels.namespace }})',
+    },
+    expr: 'appcat_probes_redis_ha_master_up{ha="true"} == 0',
+    'for': '5m',
+    labels: {
+      OnCall: '{{ if eq $labels.sla "guaranteed" }}true{{ else }}false{{ end }}',
+      runbook: 'https://kb.vshn.ch/app-catalog/how-tos/appcat/AppCatRedis.html',
+      service: serviceName,
+      severity: 'critical',
+      syn: 'true',
+      syn_team: 'schedar',
+      syn_component: 'appcat',
+    },
+  },
+  {
+    alert: 'VSHNRedisQuorumNotOk',
+    annotations: {
+      description: 'Quorum failing for 5m.',
+      summary: 'Redis quorum not OK: {{ $labels.name }} ({{ $labels.namespace }})',
+    },
+    expr: |||
+      appcat_probes_redis_ha_master_up{ha="true"} == 1
+      and on(service,namespace,name,organization,ha,sla)
+      appcat_probes_redis_ha_quorum_ok{ha="true"} == 0
+    |||,
+    'for': '5m',
+    labels: {
+      OnCall: 'false',
+      runbook: 'https://kb.vshn.ch/app-catalog/how-tos/appcat/AppCatRedis.html',
+      service: serviceName,
+      severity: 'warning',
+      syn: 'true',
+      syn_team: 'schedar',
+      syn_component: 'appcat',
+    },
+  },
+  {
+    alert: 'VSHNRedisQuorumFlapping',
+    annotations: {
+      description: 'Quorum state flipped {{ $value }} times in 10m.',
+      summary: 'Redis quorum flapping: {{ $labels.name }} ({{ $labels.namespace }})',
+    },
+    expr: |||
+      appcat_probes_redis_ha_master_up{ha="true"} == 1
```

Comment:
Shouldn't we also trigger this if it's not connected to the master? If it's flapping the chances are high it's not connecting to a master.